In [1]:
# FUZZY HOSTAGE — seluruh perhitungan memakai modul bersama.
from pathlib import Path
import sys

PROJECT_ROOT = Path.cwd()
if not (PROJECT_ROOT / "modules").is_dir():
    PROJECT_ROOT = PROJECT_ROOT.parent
if str(PROJECT_ROOT) not in sys.path:
    sys.path.insert(0, str(PROJECT_ROOT))

from modules.hostage_fuzzy import calculate_bluff_indicator, describe_bluff_level

print("Fuzzy HOSTAGE siap: hanya memakai sinyal publik dan berjalan di CPU.")

Fuzzy HOSTAGE siap: hanya memakai sinyal publik dan berjalan di CPU.


In [2]:
jumlah_tuduhan = 7
kontradiksi_klaim = 8
bukti_publik = 7
anomali_diam = 4
tekanan_vote = 6

hasil = calculate_bluff_indicator(jumlah_tuduhan, kontradiksi_klaim, bukti_publik, anomali_diam, tekanan_vote)
print(f"Indikasi bluff: {hasil:.2f}% ({describe_bluff_level(hasil).upper()})")

Indikasi bluff: 69.93% (SEDANG)


## Laporan eksekusi Fuzzy

Cell uji dijalankan dengan sinyal publik: tuduhan 7, kontradiksi 8, bukti 7, anomali diam 4, dan tekanan vote 6.

- Skor indikasi bluff: 69.93%.
- Level: sedang.
- Sesuai konteks Silent Terror: diam hanya sinyal tambahan, bukan bukti pasti.
- Skor ini sama untuk jalur SVM dan Transformer karena Fuzzy tidak memakai role asli maupun output model NLU.